src/nanovg.c

```diff
@@ -2545,11 +2545,13 @@ int nvgTextBreakLines(NVGcontext* ctx, const char* string, const char* end, floa
 					breakMaxX = 0.0;
 				}
 			} else {
-				float nextWidth = iter.nextx - rowStartX;
```

Comment:
There are two bounds on each glyph, the "logical" i.e. what the advance is based on and the the actual geometric bounds. You have to be careful to make sure you always use the same ones, else you get hard to find bugs. This code was particularly nasty to get working.

This piece of code is dealing with the "logical" units of the glyphs. Unfortunately I cannot recall all the details why I chose to do it that way, but I think it had to do with some obvious text wrapping without a good reason.

Did you change this because of some bug?